packages/components/store-notice/stories/index.stories.tsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * External dependencies
+ */
+import type { StoryFn, Meta } from '@storybook/react';
+import { NoticeBannerProps } from '@woocommerce/base-components/notice-banner';
+/**
+ * Internal dependencies
+ */
+import StoreNotice from '../';
+
+const availableStatus = [ 'default', 'success', 'error', 'warning', 'info' ];
```

Comment:
It doesn't seem useful to put this in its own variable since we're not reusing it anywhere, why not just put that in the status's options?